sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -221,10 +221,17 @@ def create_job_resources(options,  # type: PipelineOptions
                 setup_options.requirements_file, REQUIREMENTS_FILE))
         # Populate cache with packages from the requirement file option and
         # stage the files in the cache.
-        (
-            populate_requirements_cache if populate_requirements_cache else
-            Stager._populate_requirements_cache)(
-                setup_options.requirements_file, requirements_cache_path)
+        if setup_options.view_as(WorkerOptions).sdk_container_image is None:
+          # downloads the binary distributions
```

Comment:
If user passes a custom image we should advise against passing `--requirements_file` in a warning.

Reply:
Added a warning. I will add the link to the website on how to use custom containers with requirements once that website page is ready.